src/AspNet.Security.OAuth.Weixin/WeixinAuthenticationHandler.cs

```diff
@@ -30,7 +34,20 @@ public class WeixinAuthenticationHandler : OAuthHandler<WeixinAuthenticationOpti
             : base(options, logger, encoder, clock)
         {
         }
-
+        protected override async Task<HandleRequestResult> HandleRemoteAuthenticateAsync()
+        {
+            if (Options.AuthorizationEndpoint != WeixinAuthenticationDefaults.AuthorizationEndpoint)
+            {
+                var oauthstate = Request.Query["oauthstate"];
```

Comment:
`oauthstate` should be a constant, which is then used in this method and the other.